heroku/cli
# migrate(W-14170373): Upgrade pg:backups:delete

This PR migrates `pg:backups:delete` to the CLI

## Test
1. checkout this branch and run `yarn && yarn build`
2. `./bin/run pg:backups:delete --app YOUR_APP YOUR_BACKUP_ID`
3. Verify the output with the existing cli

Please note that the backwards compatible alias `pg:backups delete` was not migrated. If this is needed at a later time, `packages/pg-v5/commands/backups/index.js` will need to be migrated.
